README.md

```diff
@@ -20,341 +43,24 @@ OmniSci provides pre-built binaries for Linux for stable releases of the project
-
-The last command generates a `.tar.gz` package. The `TGZ` can be replaced with, for example, `RPM` or `DEB` to generate a `.rpm` or `.deb`, respectively.
-
-# Using
-
-The [`startomnisci`](startomnisci) wrapper script may be used to start OmniSciDB in a testing environment. This script performs the following tasks:
-
-- initializes the `data` storage directory via `initdb`, if required
-- starts the main OmniSciDB server, `omnisci_server`
-- starts the OmniSci web server, `omnisci_web_server`, for serving OmniSci Immerse
-- offers to download and import a sample dataset, using the `insert_sample_data` script
-- attempts to open OmniSci Immerse in your web browser
-
-Assuming you are in the `build` directory, and it is a subdirectory of the `omniscidb` repository, `startomnisci` may be run by:
-
-    ../startomnisci
-
-## Starting Manually
-
-It is assumed that the following commands are run from inside the `build` directory.
-
-Initialize the `data` storage directory. This command only needs to be run once.
-
-    mkdir data && ./bin/initdb data
-
-Start the OmniSciDB server:
-
-    ./bin/omnisci_server
-
-In a new terminal, start the OmniSci web server:
-
-    ./bin/omnisci_web_server
-
-If desired, insert a sample dataset by running the `insert_sample_data` script in a new terminal:
-
-    ../insert_sample_data
-
-You can now start using the database. The `omnisql` utility may be used to interact with the database from the command line:
-
-    ./bin/omnisql -p HyperInteractive
-
-where `HyperInteractive` is the default password. The default user `mapd` is assumed if not provided.
-
-You can also interact with the database using the web-based OmniSci Immerse frontend by visiting the web server's default port of `6273`:
-
-[http://localhost:6273](http://localhost:6273)
-
-Note: usage of OmniSci Immerse is governed by a separate license agreement, provided under `EULA-CE.txt`. The version bundled with this project may only be used for non-commercial purposes.
-
-# Code Style
-
-Contributed code should compile without generating warnings by recent compilers on most Linux distributions. Changes to the code should follow the [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines).
-
-## clang-format
-
-A [`.clang-format`](https://clang.llvm.org/docs/ClangFormat.html) style configuration, based on the Chromium style guide, is provided at the top level of the repository. Please format your code using a recent version (8.0+ preferred) of ClangFormat before submitting.
-
-To use:
+## Making your first contribution? 
+Check out [contribution wishlist](https://github.com/omnisci/omniscidb/contribute) with highlighted issues that would make great first contributions to this project. We also recommend visiting the [Roadmap](ROADMAP.md) to learn more about upcoming features or checking out the [Issues](https://github.com/omnisci/omniscidb/issues) to see the latest updates or proposed content.
```

Comment:
The ROADMAP.md file hasn't historically been kept up to date, so it might not be the best place to point people. Also, is the contribute page auto-generated or manual? If it's manual, you might just choose to link to a filtered issues link for easier maintainability. It's a lot easier to remember to properly tag issues than it is to update a page

Reply:
Awesome! Thanks so much for flagging these.

The Contribute wishlist is auto-generated when you add the tag "good first issue" or "help wanted".

For the Roadmap, I am working on a monthly roadmap update schedule for this. However; I included it in the wiki so we can update easier and will update the link in this right now.